src/types.d.ts

```diff
@@ -99,6 +99,7 @@ declare global {
     // Extra YAML section entries that are stringd and serve
     // to provide additional metadata about the API doc entry
     tags: Array<string>;
+    yaml_position: Position;
```

Comment:
Can you add a documentation (inline) of what this exactly is and for what it serves?